relay-cabi/include/relay.h

```diff
@@ -49,9 +49,17 @@ enum RelayDataCategory {
    */
   RELAY_DATA_CATEGORY_REPLAY = 7,
   /**
-   * A transaction that was processed but not stored.
+   * DEPRECATED: A transaction for which metrics were extracted.
+   * This category is now obsolete because the `Transaction` variant will represent
+   * processed transactions from now on.
    */
   RELAY_DATA_CATEGORY_TRANSACTION_PROCESSED = 8,
+  /**
+   * Indexed transaction events.
+   * This is the category for transaction payloads that were stored, that is, they were not
+   * filtered out by Dynamic Sampling.
```

Comment:
```suggestion
   * Indexed transaction events.
   * This is the category for transaction payloads that were stored, that is, they were not
   * filtered out by filters such as Dynamic Sampling.
```
Dynamic sampling is not the only mechanism to avoid storing transactions.

Reply:
Let me actually rewrite this. Dynamic sampling isn't really a filter -- and inbound filters would affect both data categories alike. The new text is going to be:

```
    /// Indexed transaction events.
    ///
    /// This is the category for transaction payloads that were accepted and stored in full. In
    /// contrast, `transaction` only guarantees that metrics have been accepted for the transaction.
```